docs/book/src/tasks/experimental-features/cluster-class/write-clusterclass.md

```diff
@@ -676,6 +677,102 @@ individually.
 
 </aside>
 
+## JSON patches tips & tricks
+
+JSON patches specification [RFC6902] requires that the target of
+add operation must exist.
+
+As a consequence ClusterClass authors should pay special attention when the following
+conditions apply in order to prevent errors when a patch is applied:
+
+* the patch is a JSON patch
```

Comment:
nit: I think this first point might be confusing as today we can only write JSON patches in the ClusterClass.

Future external Patches are another topic, but this document focuses a lot on writing inline paches

Reply:
The whole paragraph now is about JSON patches so this point should not be necessary. I removed it